kubernetes/samples/scripts/common/utility.sh

```diff
@@ -410,7 +410,7 @@ function createFiles {
     if [ -z $image ]; then
       sed -i -e "s|%WEBLOGIC_IMAGE%|${imageName}|g" ${dcrOutput}
     else
-      sed -i -e "s:%WEBLOGIC_IMAGE%:${image}:g" ${dcrOutput}
+      sed -i -e "s|%WEBLOGIC_IMAGE%|${image}|g" ${dcrOutput}
     fi
   else
     sed -i -e "s:%WEBLOGIC_IMAGE%:${image}:g" ${dcrOutput}
```

Comment:
What about here?

Reply:
There are a few other instances in utility.sh where a substitution takes place on WEBLOGIC_IMAGE.  Should those all be changed to use "|" instead of ":"?

Reply:
those places have been fixed with the following line for domain-home-on-pv case.
 image=$(echo ${image} | sed -e "s/\:/\\\:/g")

Reply:
👍